### PR TITLE
Add minio documentation

### PR DESCRIPTION
**Pull request by tkislan**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23567

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9267"></a>

### CODE REVIEW

**Comment by tkislan** [comment]
@MartinHjelmare Site fixed. Should be okay now.

File: source/_components/minio.markdown [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+layout: page
+title: "Minio"
+description: "Component for interacting with Minio object storage."
```
Comment (frenck) [inline source/_components/minio.markdown]:
```suggestion
description: "Integration for interacting with Minio object storage."
```

File: source/_components/minio.markdown [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+date: 2019-04-21 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```
Comment (frenck) [inline source/_components/minio.markdown]: Please provide the correct logo.

File: source/_components/minio.markdown [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+ha_category: System Monitor
+ha_iot_class: Cloud Push
+ha_release: 0.92
+ha_qa_scale: internal
```
Comment (frenck) [inline source/_components/minio.markdown]: This is not an internal component. Please remove this line.
```suggestion
```

File: source/_components/minio.markdown [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+---
+
+This component adds interaction with [Minio](https://min.io).
```
Comment (frenck) [inline source/_components/minio.markdown]:
```suggestion
This integration adds interaction with [Minio](https://min.io).
```

File: source/_components/minio.markdown [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+Also enables to listen for bucket notifications: [watch docs](https://docs.min.io/docs/minio-client-complete-guide.html#watch)
+
+To download or upload files, folders must be added to [whitelist_external_dirs](/docs/configuration/basic/).
+
+## {% linkable_title Configuration %}
+
+To enable the Minio component in your installation, add the following to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_components/minio.markdown]:
```suggestion
To enable the Minio integration in your installation, add the following to your `configuration.yaml` file:
```

File: source/_components/minio.markdown [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+{% raw %}
+```yaml
+minio:
+  host: localhost
+  port: 9000
+  access_key: SO9KNO6YT9OGE39PQCZW
```
Comment (frenck) [inline source/_components/minio.markdown]: Please use placeholders like `ACCESS_KEY` for things people need to fill out.

File: source/_components/minio.markdown [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+  secret_key: EzH5iGs3qbJr1MwIMf2AUyfqzewgd2qgrk9+i+vU
+  secure: false
+  listen:
```
Comment (frenck) [inline source/_components/minio.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — DISMISSED** [review]
Hey there @tkislan! 

I've reviewed your PR and left some comments. Could you please take a look? 👍

**Comment by klaasnicolaas** [comment]
Could you pull the latest commits from our repo and push it back to here? As far as I see, the branch that you are currently using is very outdated and we have made major changes in the meantime.

**Comment by tkislan** [comment]
Rebased and fixed deploy

**Comment by klaasnicolaas** [comment]
Ready for a merge? 😄

**Review by klaasnicolaas — APPROVED** [review]

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @tkislan! 👍

**Pull request merged**